DayTomato/DayTomato/Models/Pin.cs

```diff
@@ -77,17 +80,23 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 		{
 			JObject jo = JObject.Load(reader);
 			Pin pin = new Pin();
-
-			pin.Id = (string)jo["_id"];                             // Id of pin
-			pin.Type = (int)jo["pinType"];                          // Type of pin
-			pin.Name = (string)jo["pinName"];                       // Name of pin
-			pin.Rating = (int)jo["rating"];                         // Rating of pin
-			pin.Description = (string)jo["description"];            // Description of pin
-			pin.Likes = (int)jo["likes"];                           // Pin likes
-			pin.Latitude = (double)jo["coordinate"]["latitude"];    // Pin latitude
-			pin.Longitude = (double)jo["coordinate"]["longitude"];  // Pin longitude
-			pin.LinkedAccount = (string)jo["linkedAccount"];        // Pin linked account
-			pin.CreateDate = (DateTime)jo["createDate"];            // Pin create date
+			try
+			{
+				pin.Id = (string)jo["_id"];                             // Id of pin
```

Comment:
Will look into if we can use Json.NET's property annotations to obviate the need for this method!